.github/workflows/generate_sample.yml

```diff
@@ -0,0 +1,42 @@
+name: Generate sample project
```

Comment:
```suggestion
name: Generate a sample project
```

Reply:
Fixed in https://github.com/nimblehq/flutter-templates/pull/196/commits/be22efc9cd04ee3e2bf3bb8d95bc1e13a148e5ef